app/reducers/events.js

```diff
@@ -68,6 +68,7 @@ function mutateEvent(state: any, action: any) {
           [eventId]: {
             ...stateEvent,
             loading: false,
+            registrationCount: stateEvent.registrationCount + 1,
```

Comment:
why do you need the + 1 here? shouldn't the number sent by the server when you sign up be up to date?

Reply:
It is up to date when you fetch the event, but when you and others receive socket messages for other registrations it could be updated like this. So when you are on the calendar or event listview the registrationCount for different events will increase when others sign up. 

This gray one:
<img src="https://user-images.githubusercontent.com/5417271/32046798-4af385c2-ba44-11e7-982b-51869ea9a47f.png" width=256 />

Reply:
ah, misread - thought the registration count was sent with the websocket event. i guess that's something to look at in the future, to make sure it stays up to date even if a user doesn't receive all the websocket events?

Reply:
When I'm thinking about it we should probably cap this at 30/30, so the waiting registrations does not increase this number. Give me a second to fix that